client/knowledgebase/actions.js

```diff
@@ -0,0 +1,40 @@
+import {
+  SWITCH_TO_ARTICLE_DISPLAY,
+  SWITCH_TO_CATEGORY_DISPLAY,
+  SWITCH_TO_TOPIC_DISPLAY,
+  UPDATE_SEARCH_STRING,
+} from './constants';
+
+export const switchToArticleDisplay = (data) => {
+  return {
+    type: SWITCH_TO_ARTICLE_DISPLAY,
+    data,
+  };
+};
+
+export const switchToCategoryDisplay = (data) => {
+  return {
+    type: SWITCH_TO_CATEGORY_DISPLAY,
+    ...data, // todo
+  };
+};
+
+export const switchToTopicDisplay = (data) => {
```

Comment:
`data` is unused

Reply:
method argument removed since its not used anyway